doc/community/developer-guide.md

```diff
@@ -125,47 +126,16 @@ from sql.connection import ConnectionManager
 conn = ConnectionManager.current
 conn
 ```
+
  
-Functions that expect a `conn` (sometimes named `con`) input variable should only use connections.
+Functions that expect a `conn` (sometimes named `con`) input variable should only use connection objects.
```

Comment:
Should we open an issue to make all the `con` s into `conn` for consistency?

Reply:
yes. opened an issue https://github.com/ploomber/jupysql/issues/759